bin/gwdetchar-scattering

```diff
@@ -472,8 +476,8 @@ for i, channel in enumerate(sorted(alldata[0].keys())):
         page.tr.close()
         page.tr()
         page.th('Efficiency<br><small>(SNR&ge;8 and '
-                'f<sub>peak</sub>&lt;%.2f Hz)</small>'
-                % (2 * args.frequency_threshold))
+                '%.2f Hz</sub>&ltf<sub>peak</sub>&lt;%.2f Hz)</small>'
+                % (10,4 * args.frequency_threshold))
```

Comment:
@siddharth101, with apologies, just one more formatting thing I noticed 😄 

```suggestion
                % (10, 4 * args.frequency_threshold))
```

Reply:
:) I apologize, I really need to look up code readability.

Reply:
I recommend using the `flake8` command line tool: http://flake8.pycqa.org/en/latest/

We've been meaning to set that up to run automatically for all pull requests, it's just a matter of finding the time.

Reply:
will set this up.

Reply:
It turns out that including the list of Segments is giving the error TypeError: unhashable type: 'list'. Its probably because the way segments are being written. I will diagnose this.

Reply:
hmm, can you post the traceback?

Reply:
Traceback (most recent call last):
  File "/home/siddharth.soni/.conda/envs/gwdetchar-siddev/bin/gwdetchar-scattering", line 509, in <module>
    data.write('summary.csv')                         
  File "/home/siddharth.soni/.conda/envs/gwdetchar-siddev/lib/python3.7/site-packages/astropy/table/table.py", line 2594, in write
    io_registry.write(self, *args, **kwargs)
  File "/home/siddharth.soni/.conda/envs/gwdetchar-siddev/lib/python3.7/site-packages/astropy/io/registry.py", line 560, in write
    writer(data, *args, **kwargs)
  File "/home/siddharth.soni/.conda/envs/gwdetchar-siddev/lib/python3.7/site-packages/astropy/io/ascii/connect.py", line 86, in write_csv
    return write(table, filename, **kwargs)
  File "/home/siddharth.soni/.conda/envs/gwdetchar-siddev/lib/python3.7/site-packages/astropy/io/ascii/ui.py", line 922, in write
    writer.write(table, output)
  File "/home/siddharth.soni/.conda/envs/gwdetchar-siddev/lib/python3.7/site-packages/astropy/io/ascii/fastbasic.py", line 202, in write
    self._write(table, output, {'fill_values': [(core.masked, '')]})
  File "/home/siddharth.soni/.conda/envs/gwdetchar-siddev/lib/python3.7/site-packages/astropy/io/ascii/fastbasic.py", line 179, in _write
    writer.write(output, header_output, output_types)
  File "astropy/io/ascii/cparser.pyx", line 1067, in astropy.io.ascii.cparser.FastWriter.write
TypeError: unhashable type: 'list'

Reply:
I tried to write the table with just gpstimes and it is writing, but when i include Segments it gives error. Probably because the segments are of the form 1236602354.29375 .. 1236602354.5890625